R/namespace_linter.R

```diff
@@ -84,11 +84,7 @@ namespace_linter <- function(check_exports = TRUE, check_nonexports = TRUE) {
 
     # nocov start
     if (any(failed_namespace)) {
-      stop(
-        "Failed to retrieve namespaces for one or more of the packages used with `::` or `:::`. ",
-        "Please report the issue at https://github.com/r-lib/lintr/issues.",
-        call. = FALSE
-      )
+      cli_abort_internal("Failed to retrieve namespaces for one or more of the packages used with `::` or `:::`. ")
```

Comment:
this function is only used once, any advantage over just using `cli_abort("...", .internal=TRUE)` here?

Reply:
This function _needs_ to be internally defined for this feature to work, AFAIK.